### PR TITLE
Add Andorra Telecom Demo to the main repo

### PR DESCRIPTION
**Pull request by fmicoInfo**

### CODE REVIEW

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIAlertController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//
+
+import Foundation
+import UIKit
+import Hokusai
+
+extension Hokusai {
+    
+}
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIAlertController.swift]: remove this 🔥

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIAlertController.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+extension UIAlertController {
+
+    func colorfull() {
+        let subview = (self.view.subviews.first?.subviews.first?.subviews.first!)! as UIView
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIAlertController.swift]: 😵

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIColor.swift [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    
+    static let lightPurple = UIColor(red:0.82, green:0.02, blue:0.45, alpha:1.0)
+    static let darkPurple = UIColor(red:0.55, green:0.05, blue:0.34, alpha:1.0)
+    static let pink = UIColor(red:0.94, green:0.64, blue:0.80, alpha:1.0)
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIColor.swift]: format green:*space*0.05

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UITextFieldExtension.swift [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import UIKit
+
+@IBDesignable
+class UITextFieldExtended: UITextField {
+    
+    @IBInspectable var inset: CGFloat = 0
+    @IBInspectable var leftImage : UIImage? {
+        didSet {
+            if let image = leftImage{
+                isImageSelected = true
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UITextFieldExtension.swift]: format leftImage {

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/css/map.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
+body{
+    background: #CA1A77 url("https://www.andorratelecom.ad/o/atcomercial-theme/images/bg-pattern-footer.png") no-repeat 25% 145%;
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/css/map.css]: Can you format css and js files with VSCode or something similar?

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/js/forfet.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+function ahora(){
+    console.log('Me has ejecutaoooo!!');
+}
+
+function showLogin(){
+    console.log('Llamando!!!!');
+}
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/js/forfet.js]: 🔥

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/js/menu.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+function buildMenu() {
+    var newBody = document.createElement('body');
+    var bannerTop = document.getElementsByClassName('big-banner cont-wrapper')[0];
+    var icons = document.getElementsByClassName('item picture');
+    var titles = document.getElementsByClassName('item title');
+    var footer = document.getElementById('footer-links');
+    var contentButtonMap = footer.getElementsByTagName('h3')[1].innerText;
+    console.log(contentButtonMap);
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/js/menu.js]: 🔥

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/ViewControllers/ForfetViewController.swift [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+        
+    
+        let cancelAction: UIAlertAction = UIAlertAction(title: "cancel".localized(), style: .cancel) { action -> Void in
+            actionSheetController.dismiss(animated: true, completion: nil)
+        }
+        
+        let fullList = self.menuList.components(separatedBy: "|")
+        
+        for i in 0...fullList.count - 1 {
+            var item = fullList[i].components(separatedBy: ",")
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/ViewControllers/ForfetViewController.swift]: for section in fullList ? an old school for is not needed here

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/ViewControllers/ForfetViewController.swift [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+        default:
+            print("Any event")
+        }
+        
+    }
+    
+    func onWebLoad(_ screenlet: WebScreenlet, url: String) {
+        webScreenlet.inject(injectableScript: JsScript(name: "Nombre", js: "ahora()"))
+    }
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/ViewControllers/ForfetViewController.swift]: 🔥

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Views/CallMeBackView.swift [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+        let parameters: Parameters = [
+            "_ad_at_portal_callmeback_portlet_CallMeBackPortlet_INSTANCE_callMeBack_body": "343242",
+            "ad_at_portal_callmeback_portlet_CallMeBackPortlet_INSTANCE_callMeBack_captchaText": "1042"
+        ]
+        
+        Alamofire.request(url, headers: headers, method: .post).responseJSON{ (response) in
+            print(response)
+        }
+    }*/
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Views/CallMeBackView.swift]: 🔥

File: ios/Samples/AndorraTelecomPublic/Theme/AndorraTelecomProgressPresenter.swift [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+    /*
+    open override func hideHUDFromView(_ view: UIView?,
+                              message: String?,
+                              forInteractor interactor: Interactor,
+                              withError error: NSError?) {
+        
+        hideHud()
+        
+    } */
+
```
Comment (victorg1991) [inline ios/Samples/AndorraTelecomPublic/Theme/AndorraTelecomProgressPresenter.swift]: 🔥

**Comment by victorg1991** [comment]
Very good job Paco! 🥇  there are only some format problems and unused stuff that needs to be removed

You can install swiftlint and execute swiftlint autocorrect to fix most of the issues :)

File: ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIAlertController.swift (hunk 2 above)
Comment (fmicoInfo) [reply ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Helpers/UIAlertController.swift]: It's necessary for my solution!

**Pull request merged**